templates/store-header.php

```diff
@@ -83,7 +83,7 @@ class="profile-info-img">
                                 </li>
                             <?php } ?>
 
-                            <?php if ( ! dokan_is_vendor_info_hidden( 'email' ) && $store_user->show_email() === 'yes' ) { ?>
+                            <?php if ( ! dokan_is_vendor_info_hidden( 'email' ) && $store_user->show_email() == 'yes' ) { ?>
```

Comment:
this fix won't work. kindly rewrite this as below:
```
<?php if ( ! dokan_is_vendor_info_hidden( 'email' ) && $store_user->show_email() ) { ?>
```
note: show_email() method will return boolean value

Reply:
Done